selfdrive/locationd/calibrationd.py

```diff
@@ -165,7 +170,7 @@ def handle_cam_odom(self, trans: List[float], rot: List[float], trans_std: List[
     observed_rpy = np.array([0,
                              -np.arctan2(trans[2], trans[0]),
                              np.arctan2(trans[1], trans[0])])
-    new_rpy = euler_from_rot(rot_from_euler(self.get_smooth_rpy()).dot(rot_from_euler(observed_rpy)))
+    new_rpy: np.ndarray = euler_from_rot(rot_from_euler(self.get_smooth_rpy()).dot(rot_from_euler(observed_rpy)))
```

Comment:
There should be no reason this needs a type, isn't the euler_from_rot function typed?